leases/models.py

```diff
@@ -109,13 +110,22 @@ class BerthLease(AbstractLease):
     renew_automatically = models.BooleanField(
         verbose_name=_("renew automatically"), default=True
     )
+    _orders_relation = GenericRelation(
+        "payments.Order",
+        object_id_field="_lease_object_id",
+        content_type_field="_lease_content_type",
+    )
```

Comment:
this does not cause new migration files?

Reply:
Nope.

Reply:
gr8